packages/selfcare-v2-client/src/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from "./client.js";
-export * from "./utilities/config/config.js";
+export * from "./config/selfcareConfig.js";
```

Comment:
isn't the path for other packages `src/utilities/config.ts`?

Reply:
For end services that parse their config from `process.env`, yes - the file that parses and exports the config is under `utilities` folder.

The Selfcare does not perform the config parsing, it is a dependency of other services, the config is parsed by the end service that depends on the Selfcare. Other common utility services use the same pattern: file-manager, email-manager, etc. They are in `src/commons` and their config is defined (but not parsed) in `commons/src/config` so I used the same pattern here.

However, thinking about it, it's probably better to simply use `src/config/` folder in any config-related case, instead of `src/utilities`. Coming in a further commit!